frontend/assets/style/components/_forms.scss

```diff
@@ -219,7 +219,6 @@ textarea,
     align-items: center;
     justify-content: center;
     display: flex;
-    z-index: 1; // to be above input, always
```

Comment:
Why did you remove this z-index?

I also noticed when hovering the selectcombo the icon disappears:
![image](https://user-images.githubusercontent.com/14869087/76437372-626ea100-63b1-11ea-9177-9d1acd2f4cbb.png)

Please verify

Reply:
I didn't remove the z-index, I **increase** the z-index. After I pull the last branch from master lots of element started to be hidden like this...

Reply:
I didn't remove the z-index, I **increase** the z-index. After I pull the last branch from master lots of element started to be hidden like this...